ios/brave-ios/Sources/Brave/Frontend/Browser/BrowserViewController/BVC+ToolbarDelegate.swift

```diff
@@ -218,7 +218,16 @@ extension BrowserViewController: TopToolbarDelegate {
       hideSearchController()
     } else {
       showSearchController()
-      searchController?.setSearchQuery(query: text)
+
+      Task {
+        await searchController?.setSearchQuery(
+          query: text,
+          showSearchSuggestions: URLBarHelper.shared.shouldShowSearchSuggestions(
+            using: topToolbar.locationLastReplacement
+          )
+        )
+      }
+
       searchLoader?.query = text.lowercased()
```

Comment:
Is `searchLoader?.query = text.lowercased()` completely unrelated to setting the search query above? Does it in particular need to wait until after the `setSearchQuery` is called?

Reply:
It has nothing to do with this change, this is populating the bookmarks, history etc for different section of suggestions